### PR TITLE
[2.7] Bug 537795

### PR DESCRIPTION
**Pull request by jgrassel**
for #212
https://bugs.eclipse.org/bugs/show_bug.cgi?id=537795

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=537795

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=537795

File: jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/query/TestQueryCase.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation. All rights reserved.
```
Comment (lukasj) [inline jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/query/TestQueryCase.java]: Wrong license and license header as such. Use the template from [the Sample](https://github.com/eclipse-ee4j/eclipselink/blob/master/project-admin/Sample.java), please.

**Comment by dazey3** [comment]
I am going to put all my review comments in the PR for master. Once that is committed, then I expect the 2.7 PR to be updated and then Ill approve as well

**Comment by jgrassel** [comment]
Cancelling, going with the fix in pull request 216.

**Pull request closed without merging**